owl/EASE.owl

```diff
@@ -1126,14 +1055,6 @@ Such an event does NOT fall under the definition of Accident here. An example of
     
 
 
-    <!-- http://www.ease-crc.org/ont/EASE.owl#Location -->
```

Comment:
What's the current plan about ontologically describing locations?

Reply:
Please have a look in EASE-OBJ, the concept should have moved there.

Qualitatively, objects may have a *Localization* which is a quality, it takes values only from *SpaceRegion*. Currently, we have 6DPose and 3DPosition underneath *SpaceRegion*. This is not yet very fine-grained. I struggled a bit with getting a more fine grained taxonomy below it, but maybe you can give it a try at some point! For now, I was thinking it is sufficient to have this shallow taxonomy.

As *SpaceRegion*, KnowRob will rather store the data into mongo DB instead of having it in the triple store. This allows for changing the regions of a quality over time without having to deal with problems regarding representing time in OWL.

Reply:
Oh and regarding quaternions: I doubt that we will ever qualify quaternion components, so I removed the class in favour of the dataproperty *has orientation vector*